content/Hello yaaf - Juan Manuel Ramallo.md

```diff
@@ -0,0 +1,73 @@
+## Yet another active form
+
+Introducing [yaaf](https://github.com/rootstrap/yaaf), a gem to ease the usage of the form object pattern in rails apps.
+
+### ~The~ My pain points
+
+The form object pattern is widely used across Rails apps,
+and yet we tend to write these form objects in a different manner from project to project.
+
+Most of the time, they don't even have the same interface across form objects in the same app.
+
+We also tend to write validations twice, in the model and in the form object,
+just so that the form object has the errors and the view can display them.
+
+Well, I can say that most of our in-house form objects make well use of database transactions.
```

Comment:
Here I would change the word well for _good_

Reply:
I don't think they mean the same thing. With "Well" I meant to create a pause before agreeing with the reader on a matter, the matter is that "all existing form objects are using database transactions correctly"

Reply:
I don't get the pause sense there. I agree with @juan-apa, `well` sounds odd for me too.

Reply:
Ok, two disagreements. I'll change. 

What do you think of removing it altogether? 
```suggestion
I can say that most of our in-house form objects make well use of database transactions.
```

Reply:
I believe they meant to change the "make _well_ use of" to "make _good_ use of", or at least thats what I would change here. 😆I do get what you want to achieve with the first well.

Reply:
hahahahha good catch nacho! thanks!

```suggestion
Well, I can say that most of our in-house form objects make good use of database transactions.
```
Like this then?